src/backend/aspen/api/authz.py

```diff
@@ -219,21 +221,25 @@ def require_group_privilege(
 
 
 class AuthorizedRow:
-    def __init__(self, privilege: str, model: idbase):
+    def __init__(self, privilege: str, model: idbase, id_field: Optional[str] = None):
         self.privilege = privilege
         self.model = model
+        if not id_field:
+            id_field = "row_id"
+        self.id_field = id_field
 
     async def __call__(
         self,
-        row_id: int,
+        request: Request,
         auth_context: AuthContext = Depends(get_auth_context),
         session: AsyncSession = Depends(get_db),
     ) -> idbase:
         authz_session = AuthZSession(session, auth_context)
         query = await authz_session.authorized_query(self.privilege, self.model)
+        id_value = int(request.path_params[self.id_field])
```

Comment:
I should have caught this when reviewing, my bad.

Reply:
nah, I was intentionally trying to standardize everything to `row_id` at the time but just looks like I took it toooo far.